src/steps/emacs.rs

```diff
@@ -21,7 +21,11 @@ pub struct Emacs {
 impl Emacs {
     fn directory_path(base_dirs: &BaseDirs) -> Option<PathBuf> {
         #[cfg(unix)]
-        return base_dirs.home_dir().join(".emacs.d").if_exists();
+        return base_dirs
+            .home_dir()
+            .join(".emacs.d")
+            .if_exists()
+            .or_else(|| base_dirs.config_dir().join("emacs").if_exists());
```

Comment:
`base_dirs.config_dir()` returns `~/Library/Preferences` in macOS. I'm not sure this is where Emacs 27 expects to find its configuration in macOS.

Reply:
Ah indeed, looking at the code emacs checks for the existence of the `$XDG_CONFIG_HOME` environment variable and, in case it doesn't exist, falls back to `~/.config/` (which would probably be the case for macOS).  Apologies for that.

I've force pushed a change that should sort this out (although this feels like a hack to me, so feel free to suggest something more idiomatic).

Reply:
Seems reasonable, but you should also check if the variable `XDG_CONFIG_HOME` itself exists.

Reply:
You mean on macOS?  Right, I thought these env variables were just not a thing on macOS...

I've pushed something that should (hopefully) work